10.2/Dockerfile

```diff
@@ -21,9 +21,10 @@ RUN set -ex; \
 # https://github.com/tianon/gosu/releases
 ENV GOSU_VERSION 1.12
 RUN set -eux; \
-	savedAptMark="$(apt-mark showmanual)"; \
 	apt-get update; \
-	apt-get install -y --no-install-recommends ca-certificates wget; \
+	DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends ca-certificates apt-transport-https; \
```

Comment:
Oh, that being said, this one's an eyebrow-raiser.  Do you actually need `apt-transport-https`?  (New enough APT does not require it anymore as the functionality is instead bulit-in.)

We also frown on `DEBIAN_FRONTEND=noninteractive` (because it's really a bug in `debconf` that it's ever necessary), but when used in this limited-scope way it's not an explicit blocker for us.

Reply:
(Organizationally, it's also strange to install a couple persistent dependencies in this block when there's already an explicit "install a few persistent dependencies" block above.)

Reply:
`apt-transport-https`, quite right not needed - will remove. Main point of two apt-get installs was so that `ca-certificates` was installed here and wasn't removed when it was needed later for the apt download of `https://ftp.osuosl.org`.

Yes `debconf` continuing when no `TERM` set and when `isatty` would fail is indeed frown worthy. I was also in the "pragmatic limited-scope and not to happy about it" thoughts too.